app/components/admin/all_collections_component.html.erb

```diff
@@ -1,10 +1,9 @@
 <section class="mb-5 allCollections">
-  <%= form_with url: search_path, method: :get, local: false,
-                data: { controller: 'find-druid', type: 'html', action: 'ajax:error->find-druid#onFailure' },
+  <%= form_with url: admin_druid_searches_path, method: :get,
```

Comment:
I think the idea was to remove the form from here.

Reply:
Thanks, I hadn't understood that since it was in older mockups. Will remove from that location.

Reply:
Removed the search box.